src/app/AuthRoute.tsx

```diff
@@ -29,7 +29,7 @@ const AuthRoute = ({ children, ...rest }: AuthRouteProps): JSX.Element => {
 						id="ipc-title"
 						textAlign="center"
 					>
-						Inter Planetary Cloud
+						Inter Planetary Cloud / Computing
```

Comment:
Isn't the project named Inter Planetary Cloud with a Storage and a Computing part ?

The name should then be just Inter Planetary Cloud, or Inter Planetary Storage / Computing imho.

Reply:
> Isn't the project named Inter Planetary Cloud with a Storage and a Computing part ?
> 
> The name should then be just Inter Planetary Cloud, or Inter Planetary Storage / Computing imho.

I hadn't seen it like that. But suddenly you might as well keep the name as it was with InterPlanetaryCloud which contains a Storage part and another Computing part.